Makefile.ci

```diff
@@ -24,9 +24,7 @@ CI_TARGETS=ci-all \
 
 # Generic rule, we use make to easy travis integraton with mixed rules
 $(CI_TARGETS): ci-%:
-	rm -f ci-$*.ok
-	+(./dev/ci/ci-$*.sh 2>&1 && touch ci-$*.ok) | tee time-of-build.log
+	+./dev/ci/ci-pipe-tee.sh ./dev/ci/ci-$*.sh time-of-build.log
 	echo 'Aggregating timing log...' && echo -en 'travis_fold:start:coq.test.timing\\r'
 	python ./tools/make-one-time-file.py time-of-build.log
 	echo -en 'travis_fold:end:coq.test.timing\\r'
```

Comment:
Yes, this is desired; if the script fails, we don't care much about how long various files took, and we just want to see the error message it failed with.  This was a point of confusion, previously, because the original make error message was hard to discover.  See https://github.com/coq/coq/pull/956#issuecomment-321206221

You mean, `ci-wrapper.sh` does the echo and the python, etc?  I could do that...